### PR TITLE
Expand content on confirmation page

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Expand content on confirmation page. Keep 6 column layout for xl media query.

Create override layout componentn for confirmation page.

## Justify why they are needed

Let's see if we really need a special layout for this page in the future.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1595** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1595" target="_blank">Graphite</a> 
    * **PR #1596** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1596" target="_blank">Graphite</a> 
      * **PR #1597** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1597" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1597?utm_source=stack-comment).

File: apps/store/src/components/ConfirmationPage/FooterSection.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const FooterImage = styled(Image)({
   objectFit: 'contain',
   objectPosition: 'right bottom',
 
-  maxWidth: '1920px',
+  maxWidth: '120rem',
```
Comment (guilhermespopolin) [inline apps/store/src/components/ConfirmationPage/FooterSection.tsx:41]: Maybe we should update [`GridLayout`](https://github.com/HedvigInsurance/racoon/blob/main/apps/store/src/components/GridLayout/GridLayout.tsx#L10) to use `rem`s as well.

**Review by guilhermespopolin — APPROVED** [review]
🆗

File: apps/store/src/components/ConfirmationPage/FooterSection.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/ConfirmationPage/FooterSection.tsx:41]: aha didn't know we used px there

**Pull request merged**